src/main/scala/fi/aalto/cs/apluscourses/intellij/utils/ModuleUtils.scala

```diff
@@ -34,4 +36,66 @@ object ModuleUtils {
     nonEmpty
   }
 
+  def naiveValidate(@NotNull command: String): Boolean =
+    command.matches("import\\so1\\.[a-z]*(\\_|\\._)$")
+
+  def clearCommands(@NotNull imports: Array[String]): Array[String] =
+    imports
+      .clone
+      .map(_.replace("import ", ""))
+      .map(_.replace("._", ""))
```

Comment:
Why?

Reply:
well, the actual commands are stored as full commands (and being written to the file as well), however, in the REPL, the message looks like this:
```bash
Welcome to Scala 2.13.1 (OpenJDK 64-Bit Server VM, Java 11.0.7).
Loaded A+ Courses module [Train]. Auto-imported packages [o1, o1.train] for your convenience.
Write a line (or more) of Scala and press [Ctrl+Enter] to run it. Use [Up] and [Down] to scroll through 
your earlier inputs. Changes to the module are not loaded automatically. If you edit the files, restart 
the REPL with [Ctrl+F5] or the icon on the left. 
Type in expressions for evaluation. Or try :help.
```
(contains only the names of the imported packages). the only purpose is to have this text nice-lookin'. safe for now, as the regex you've noticed earlier will allow only imports, regardless of what someone can put into the course config. thus, not sure if it's 100% bulletproof (but at least foolproof it is :wink: )

Reply:
Oh, yes sure, that makes sense.